tests/integration/test_app_create.py

```diff
@@ -89,8 +89,8 @@ def test_adds_logging_of_span_and_trace(self):
             }
             application.test_client().get("/", headers=x_cloud_headers)
 
-            self.assertEqual(1, logger.bind.call_count)
-            _, kwargs = logger.bind.call_args
+            self.assertEqual(2, bind_contextvars.call_count)
```

Comment:
How come the count increased?.

Reply:
We replaced `logger.new(request_id=request_id)` we do `contextvars.bind_contextvars(request_id=request_id)` now.

Reply:
How come that increases the count?.

Reply:
it calls `bind_contextvars()` twice now in setup and then when making a request?  before it was `bind()` once and `new()` once?

Reply:
and we only counted calls of `logger.bind()`?

Reply:
Okay, so there is no replacement for `logger.new`?. Is that deprecated?. Is there some doc that mentions this?.

Reply:
[doc](https://www.structlog.org/en/stable/api.html#structlog.BoundLoggerBase.new), it clears context and binds initial_values using [bind](https://www.structlog.org/en/stable/api.html#structlog.BoundLoggerBase.bind) so that's why we had to remove it, cause w don't use the old `.bind()` anymore.